astropy/cosmology/io/_builtin/mapping.py

```diff
@@ -38,8 +38,9 @@
 
     >>> cosmo = Cosmology.from_format(cm, format="mapping")
     >>> cosmo
-    FlatLambdaCDM(name="Planck18", H0=67.66 km / (Mpc s), Om0=0.30966,
-                  Tcmb0=2.7255 K, Neff=3.046, m_nu=[0. 0. 0.06] eV, Ob0=0.04897)
+    FlatLambdaCDM(name='Planck18', H0=<Quantity 67.66 km / (Mpc s)>, Om0=0.30966,
+                  Tcmb0=<Quantity 2.7255 K>, Neff=3.046,
+                  m_nu=<Quantity [0.  , 0.  , 0.06] eV>, Ob0=0.04897)
```

Comment:
Renaming modules should not cause `FlatLambdaCDM.__repr__()` to produce different output, and it looks like `pytest` agrees with me. This is a large pull request, I'd rather not review it until it has been cleaned up.

Reply:
All these docstrings weren't being tested before. Now they are!

Reply:
I'm trying to identify the change that caused all these docstrings to start being tested. But regardless of the cause, I can confirm that  the old docstrings are wrong, the new ones are correct, and that the docstrings are now being tested.

Reply:
The output here depends on the Python version, see https://github.com/astropy/astropy/blob/dde34be8684874c8b242bcd33440418617489b9b/pyproject.toml#L204-L207
There's no need to update the docstrings in this pull request.

Reply:
Ah. And since `_io` is now `io`...
I can separate this into 2 PRs, first for the re-org. Then for fixing the docstrings.

Reply:
Docstring difference looks like `repr` vs `str` on the quantities. Not sure `repr` is better here.

Reply:
>>> calls repr. We would have to do print for everything.

Maybe Quantity's repr should be updated to call repr on its contents and enable `eval(repr(q))`

Reply:
I don't like that `eval(repr(q))` doesn't work, but admittedly for arrays this is always going to be a problem (at least large ones). And `ndarray` doesn't produce a repr that can be evaluated either... This is why I was wondering if it really made sense to use `repr(q)` in cosmology.

Reply:
@eerovaher, done! removed the doc edits to mapping, row, and table.